pumps/mongo_selective.go

```diff
@@ -244,7 +244,7 @@ func (m *MongoSelectivePump) AccumulateSet(data []interface{}) [][]interface{} {
 		if thisItem.ResponseCode == -1 {
 			continue
 		}
-		sizeBytes := len([]byte(thisItem.RawRequest)) + len([]byte(thisItem.RawRequest))
+		sizeBytes := len([]byte(thisItem.RawRequest)) + len([]byte(thisItem.RawResponse)) + 1024
```

Comment:
could you explain why + 1024?

Reply:
@sredxny because 1kb is the metadata average size, as shown [here](https://github.com/TykTechnologies/tyk-pump/blob/fac0819107b34c92e56d0423e8c5b90feaed6e8d/pumps/mongo.go#L561)

Reply:
@mativm02 add a code comment for that :)